README.md

```diff
@@ -3,14 +3,63 @@
 
 Add an email address for use in your [IRMA app](https://github.com/privacybydesign/irma_mobile).
 
-
 ## Setting up the server
 
-1. Generate JWT keys for the issuer
+### Prerequisite
+
+Go, Gradle v4, JDK, Yarn, and an installation of IRMA server:
+
+```bash
+git clone git@github.com:privacybydesign/irmago.git
+cd irmago/
+go install ./irma
```

Comment:
You can also do this in a oneliner without a manual clone `go install github.com/privacybydesign/irmago@latest`